org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/RecordsFilter.java

```diff
@@ -85,7 +85,8 @@ boolean isFieldAccessor(final MethodNode m) {
 			if (!((FieldInsnNode) cursor).name.equals(m.name))
 				return false;
 			next();
-			if (cursor == null) return false;
+			if (cursor == null)
+				return false;
```

Comment:
We always use braces for blocks. Can you please add them here and above?

```suggestion
			if (cursor == null) {
				return false;
			}
```

Reply:
Fixed